mods/common/languages/en.ftl

```diff
@@ -172,5 +283,289 @@ none = None
+rename-replay-accept = Rename
+
+delete-replay-title = Delete selected replay?
+delete-replay-prompt = Delete replay { $replay }?
+delete-replay-accept = Delete
+
+delete-all-replays-title = Delete all selected replays?
+delete-all-replays-prompt =
+    { $count ->
+        [one] Delete { $count } replay.
+       *[other] Delete { $count } replays.
+    }
+delete-all-replays-accept = Delete All
+
+replay-deletion-failed = Failed to delete replay file '{ $file }'. See the debug.log file for details.
+
+## ReplayUtils
+incompatible-replay-title = Incompatible Replay
+incompatible-replay-prompt = Replay metadata could not be read.
+-incompatible-replay-recorded = It was recorded with an
+incompatible-replay-unknown-version = { -incompatible-replay-recorded } unknown version: { $version }.
+incompatible-replay-unknown-mod = { -incompatible-replay-recorded } unknown mod: { $mod }.
+incompatible-replay-unavailable-mod = { -incompatible-replay-recorded } unavailable mod: { $mod }.
+incompatible-replay-incompatible-version = { -incompatible-replay-recorded } incompatible version: { $version }.
+incompatible-replay-unavailable-map = { -incompatible-replay-recorded } unavailable map: { $map }.
+
+## ServerCreationLogic
+internet-server-nat-A = Internet Server (UPnP/NAT-PMP
+internet-server-nat-B-enabled = Enabled
+internet-server-nat-B-not-supported = Not Supported
+internet-server-nat-B-disabled = Disabled
+internet-server-nat-C = ):
+
+local-server = Local Server:
+
+server-creation-failed-prompt = Could not listen on port { $port }
+server-creation-failed-port-used = Check if the port is already being used.
+server-creation-failed-error = Error is: "{ $message }" ({ $code })
+server-creation-failed-title = Server Creation Failed
+server-creation-failed-cancel = Back
+
+## ServerListLogic
+players-online =
+    { $players ->
+        [one] { $players } Player Online
+       *[other] { $players } Players Online
+    }
+
+search-status-failed = Failed to query server list.
+search-status-no-games = No games found. Try changing filters.
+no-server-selected = No Server Selected
+
+map-status-searching = Searching...
+map-classification-unknown = Unknown Map
+
+players-label =
+    { $players ->
+        [zero] No Players
+        [one] One Player
+       *[other] { $players } Players
+    }
+
+bots-label =
+    { $bots ->
+        [zero] No Bots
+        [one] One Bot
+       *[other] { $bots } Bots
+    }
+
+## ServerListLogic, ReplayBrowserLogic, ObserverShroudSelectorLogic
 players = Players
+
+## ServerListLogic, GameInfoStatsLogic
+spectators = Spectators
+spectators-label =
+    { $spectators ->
+        [zero] No Spectators
+        [one] One Spectator
+       *[other] { $spectators } Spectators
+    }
+
+## ServerlistLogic, GameInfoStatsLogic, ObserverShroudSelectorLogic, SpawnSelectorTooltipLogic, ReplayBrowserLogic
+team-no-team =
+    { $team ->
+        [zero] No Team
+       *[other] Team { $team }
+    }
+
```

Comment:
This is a little hacky way of using plural where we're not really counting anything - it's just the team numeric ID. I can see how it saves us from creating separate messages so I guess we can let it slide :thinking: Perhaps add a comment to explain it?

Reply:
On second thought - these should be separate messages (e.g. 'no-team-name' and 'team-name'). Using the [zero] variant here is wrong not just semantically but because there may be languages that don't use the [zero] case at all.

Reply:
Addressed in #20207

Reply:
Maybe I was too desperate to find a good use case for Fluent selectors.